observer/observer.go

```diff
@@ -43,18 +43,21 @@ func (o *Observer) processBlock(events chan<- Event, block *blockatlas.Block) {
 	}
 
 	// Emit events
-	emitted := make(map[string]bool)
+	emitted := make(map[string]blockatlas.Direction)
 	platform := bitcoin.UtxoPlatform(o.Coin)
 	for _, sub := range subs {
+
 		txs := txMap[sub.Address].Txs()
 		for _, tx := range txs {
-			if _, ok := emitted[tx.ID]; ok {
-				continue
-			}
+
 			xpub, _ := o.Storage.GetXpubFromAddress(o.Coin, sub.Address)
-			if len(xpub) != 0 {
```

Comment:
Could you out utxo handling

Reply:
Is the same verification, but we don't match the negative numbers